src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/OssSecurityScoreTest.java

```diff
@@ -21,4 +44,31 @@ public void serializeAndDeserialize() throws IOException {
     assertEquals(score, clone);
```

Comment:
Do we need a test to verify if the Used Values are properly de-/serializing?

Reply:
Yeah, we have `ScoreValueTest.serializeAndDeserialize()`.

Reply:
👍